code/game/objects/items/weapons/tools.dm

```diff
@@ -273,6 +273,24 @@
 		else
 			user << "<span class='notice'>You are already trying to pull out a teeth!</span>"
 		return
+	if(ishuman(C) && C.dna.species.id == "wire" && C != user)

```

Comment:
dude they're made of wire what do you think happens when you cut them up into little pieces with wirecutters?